utilities/ft_hastoolbox.m

```diff
@@ -388,6 +389,8 @@
     dependency = {'fig2plotly' 'savejson'};
   case 'JSONIO'
     dependency = {'jsonread', 'jsonwrite', 'jsonread.mexa64'};
+  case 'CPD' 
+    dependency = {'cpd'};
```

Comment:
if possible add some more functions from the toolbox to the list. This one function is not very distinctive.

Reply:
I added more functions from the toolbox that are necessary for fittemplate